### ISSUE TITLE
Pull Request Semantics

### ISSUE THREAD

**Issue by Scott-Young-6746** [Mokane3562/PokerBot-kt-Gradle#2]
It would be a good idea for no contributor to push directly to master (or CleanCode if that's what we use). Instead each contributor should make a PR and another contributor should accept that PR, after discussion if necessary.

**Comment by Mokane3562** [comment]
@Scott-Young-6746 Agreed. We will create branches and have the reviewer merge them into master. If you know of a way to enforce and document this, please do so.
